x-pack/plugins/observability_onboarding/public/components/app/custom_logs/wizard/select_logs.tsx

```diff
@@ -0,0 +1,241 @@
+          <EuiFlexGroup>
+            <EuiFlexItem grow={false} style={{ width: '50%' }}>
+              <OptionCard
+                title={i18n.translate(
+                  'xpack.observability_onboarding.selectLogs.streamLogFiles',
+                  {
+                    defaultMessage: 'Stream log files',
+                  }
+                )}
+                iconType="desktop"
+                onClick={() => {
+                  setState({ ...getState(), logsType: 'log-file' });
+                  goToStep('configureLogs');
+                }}
+                isSelected={false}
+                description={i18n.translate(
+                  'xpack.observability_onboarding.selectLogs.streamLogFiles.description',
+                  {
+                    defaultMessage: 'Ingest arbitrary custom log files.',
+                  }
+                )}
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </LogsTypeSection>
+        <EuiHorizontalRule margin="l" />
+        <LogsTypeSection title="Network streaming logs">
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <OptionCard
+                title={i18n.translate(
+                  'xpack.observability_onboarding.selectLogs.sysLog',
+                  {
+                    defaultMessage: 'Syslog logs',
+                  }
+                )}
+                iconType="documents"
+                onClick={() => {}}
+                isSelected={false}
+                description={i18n.translate(
+                  'xpack.observability_onboarding.selectLogs.sysLog.description',
+                  {
+                    defaultMessage:
+                      'Collect raw log data from listening ports.',
+                  }
+                )}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <OptionCard
+                title={i18n.translate(
+                  'xpack.observability_onboarding.selectLogs.httpEndpointLogs',
+                  {
+                    defaultMessage: 'HTTP Endpoint logs',
+                  }
+                )}
+                iconType="documents"
+                onClick={() => {}}
+                isSelected={false}
+                description={i18n.translate(
+                  'xpack.observability_onboarding.selectLogs.httpEndpointLogs.description',
+                  {
+                    defaultMessage:
+                      'Collect JSON data from listening HTTP port.',
+                  }
+                )}
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="m" />
+        </LogsTypeSection>
+        <EuiSpacer size="m" />
+        <EuiLink
+          href="#"
+          target="_blank"
+          onClick={(event: MouseEvent) => {
+            event.preventDefault();
+            navigateToAppUrl('/integrations/browse/observability');
+          }}
+        >
+          Explore other integrations
```

Comment:
this needs i18n

Reply:
Good catch! Thanks

Reply:
Addressed in [2826d0f](https://github.com/elastic/kibana/pull/156182/commits/2826d0f297aef033a32f9ea6675aea86ce7f5d48).